pkg/storaged/fsys-panel.jsx

```diff
@@ -142,7 +142,7 @@ export class FilesystemsPanel extends React.Component {
                             title: <StorageUsageBar stats={[Number(fs.Used[0] && Number(fs.Used[1])),
                                 Number(pool.TotalPhysicalSize)]}
                                                     critical={1} total={total} offset={offsets[i]} />,
-                            props: { className: "pf-u-text-align-right" }
+                            props: { className: "pf-v5-u-text-align-right" }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230519-134728-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)